decidim-core/app/views/decidim/reported_mailer/report.html.erb

```diff
@@ -4,6 +4,47 @@
   <%= t(".report_html", { url: reported_content_url }) %>
 </p>
 
-<p>
-  <%= link_to t(".manage_moderations"), manage_moderations_url %>
-</p>
+<br>
+
+<p><b><%= t(".date") %></b></p>
+<p><%= l @report.created_at, format: :short %></p>
+
+<p><b><%= t(".participatory_space") %></b></p>
+<p><%= translated_attribute @participatory_space.title %></p>
+
+<p><b><%= t(".reason") %></b></p>
+<p><%= t(@report.reason, scope: "decidim.shared.flag_modal") %></p>
+
+<% if @report.details.present? %>
+  <p><b><%= t(".details") %></b></p>
+  <blockquote>
+    <%= @report.details %>
+  </blockquote>
+<% end %>
+
+<% if @content.present? %>
+  <p><b><%= t(".content") %></b></p>
+  <blockquote>
+    <h5><%= translated_attribute @content[:title] %></h5>
+    <p><%= translated_attribute @content[:body] %></p>
+  </blockquote>
+<% end %>
+
+<p><b><%= t(".authors") %></b></p>
+<p><%= link_to @author.name, author_profile_url, target: :blank %></p>
```

Comment:
This should include a safe guard (probably in combination with `reporter_mailer.rb`) in case the `@author` is not present or is not a normal user.

For instance, official proposal have the author as the organization itself, which does not have nickname (called in the `author_profile_url` method). In these case this error is thrown:

```
ActionView::Template::Error (undefined method `nickname' for #<Decidim::Organization:0x00007f3515117e10>):
```

Reply:
Done! I just [have to fix this](https://github.com/decidim/decidim/pull/6053#issuecomment-635398210)